wps-common/src/main/java/au/org/aodn/aws/wps/status/WpsConfig.java

```diff
@@ -156,7 +156,8 @@ private String getEncryptedEnvironmentVariable(String keyName) {
      * @param keyValue
      * @return Decrypted key value.
      */
-    private String decryptKey(String keyValue) {
+    private String decryptKey(String keyValue)
+    {
```

Comment:
We need to agree on the formatting style, so that it is uniform across the code.